web/sites/default/config/facets.facet.filter_by_service_area.yml

```diff
@@ -0,0 +1,73 @@
+uuid: bda2b240-306a-4372-a382-755d07b5f048
+langcode: en
+status: true
+dependencies:
+  config:
+    - search_api.index.full_index
+  module:
+    - search_api_page
+id: filter_by_service_area
+name: 'Filter by Service Area'
+weight: 1
+min_count: 1
+missing: false
+missing_label: others
+url_alias: service-area
+facet_source_id: 'search_api:search_api_page__search_portland_gov'
```

Comment:
The facet source is on the site search but the block is only visible on the Bureau/office search view. Please make the source as "search_api:views_page__sitewide_groups_index__page_offices"